src/middleware/aws.rs

```diff
@@ -0,0 +1,98 @@
+use opentelemetry::{
+    global,
+    trace::{SpanKind, Tracer},
+};
+pub use opentelemetry::{trace::Span, KeyValue};
+pub use opentelemetry_semantic_conventions as semcov;
```

Comment:
Why are those imports public?

Reply:
I found where `Span` and `KeyValue` are used, but we should probably do as I suggested in https://github.com/nentgroup/profiles/pull/183/files#r1499735072 instead

Reply:
Then we could drop the `pub` here